factory/alchemy.py

```diff
@@ -44,8 +56,20 @@ class Meta:
     def _create(cls, model_class, *args, **kwargs):
         """Create an instance of the model, and save it to the database."""
         session = cls._meta.sqlalchemy_session
+        session_persistence = cls._meta.sqlalchemy_session_persistence
+        if cls._meta.force_flush:
+            session_persistence = SESSION_PERSISTENCE_FLUSH
+            warnings.warn("force_flush has been deprecated as of 2.8.0 and will be removed in 3.0.0. Please use `sqlalchemy_session_persistence='flush'` insead.", DeprecationWarning)
```

Comment:
It's better to set the ``warnlevel`` on such warnings, to make sure that the warning references the actual code.

However, putting an accurate warning is kinda complicated, I might need to adjust that in the general ``OptionDefault`` class instead to raise warning directly at the metaclass declaration level.